src/js/components/Accordion.js

```diff
@@ -39,7 +39,7 @@ export default class Accordion extends Component {
     const accordionChildren = React.Children.map(children, (child, index) => {
       return React.cloneElement(child, {
         id: 'accordion-panel-' + index,
-        active: !openMulti ? (this.state.activeIndex === index) : null,
+        active: !openMulti ? (this.state.activeIndex === index && child.props.active) : child.props.active,
```

Comment:
I believe the right check should be:

`active: !openMulti ? (this.state.activeIndex === index) : child.props.active,`